.gitignore

```diff
@@ -109,6 +109,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+ecommerce/.env
```

Comment:
according to documentation below, .env file should be at the root of the project so maybe gitignore file should ignore this path directly as `.env`

![image](https://user-images.githubusercontent.com/27494848/143658322-a51185cf-5db9-45b5-9b1c-bc36b2e1190f.png)

Reply:
You are right. Documentation is clear. I missed that part. Thanks